docs/blog/infrastructure-as-a-code-startup.md

```diff
@@ -0,0 +1,162 @@
+﻿## **Infrastructure as a Code** Startup Solution
```

Comment:
What does startup mean here?

Reply:
I think it is a little vague. Better words may be Setting up or initiation?

Reply:
Just remove startup (":

Reply:
OK